BROWSER-SUPPORT.md

```diff
@@ -1,7 +1,5 @@
 # Browser support
 
-## Introduction
-
 The UI-Kit team needs to support all of our users, regardless of their device, web browser or other user agent.
 
 Alongside accessibility, extensive browser and device support is a legal requirement under the <a href="https://www.legislation.gov.au/Latest/C2016C00763" rel="external">Disability Discrimination Act (1992)</a>, requiring equal access to information on laws and government programs.
```

Comment:
What about moving the reference to the Act to the README.md in the meantime? I can rework the text here to link back to it. How about have a chat tomorrow? @klepas